package.json

```diff
@@ -8,140 +8,45 @@
   },
   "scripts": {
     "lint": "solium --dir ./contracts",
-    "build": "webpack -p",
-    "truffle": "./node_modules/.bin/truffle",
     "test": "truffle migrate && truffle test --network test",
-    "dev": "webpack-dev-server -d --inline --hot",
     "coverage": "./node_modules/.bin/solidity-coverage",
     "build-contracts": "sol-merger './contracts/*.sol' ./build",
     "solgraph": "solgraph contracts/Conference.sol  > doc/Conference.dot && dot -Tpng doc/Conference.dot -o doc/Conference.png"
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/makoto/blockparty.git"
+    "url": "git+https://github.com/noblocknoparty/blockparty.git"
   },
   "author": "",
   "license": "ISC",
   "bugs": {
-    "url": "https://github.com/makoto/blockparty/issues"
+    "url": "https://github.com/noblocknoparty/blockparty/issues"
   },
-  "homepage": "https://github.com/makoto/blockparty#readme",
+  "homepage": "https://github.com/noblocknoparty/blockparty#readme",
   "dependencies": {
     "@ensdomains/ens": "^0.1.1",
     "axios": "^0.17.1",
-    "babel-core": "^6.8.0",
-    "babel-loader": "^6.2.4",
     "babel-polyfill": "^6.23.0",
-    "babel-preset-es2015": "^6.6.0",
-    "babel-preset-latest": "^6.24.1",
-    "babel-preset-react": "^6.5.0",
-    "bluebird": "^3.3.5",
-    "copy-webpack-plugin": "^3.0.0",
-    "crypto-browserify": "^3.11.1",
-    "css-loader": "^0.23.1",
     "eth-ens-namehash": "^2.0.8",
     "ethereum-ens": "^0.7.4",
     "ethereum-input-data-decoder": "0.0.3",
     "etherscan-api": "^3.0.11",
-    "event-emitter": "^0.3.4",
-    "file-loader": "^0.8.5",
-    "jquery": "^3.3.1",
-    "json-loader": "^0.5.4",
-    "material-ui": "^0.15.2",
-    "mathjs": "^3.2.1",
     "moment": "^2.18.1",
```

Comment:
Do we still need moment?

Reply:
It's used in scripts and `test/stress.js`